tools/crm_mon_curses.c

```diff
@@ -429,7 +429,7 @@ cluster_maint_mode_console(pcmk__output_t *out, va_list args) {
         curses_formatted_printf(out, "\n              *** Resource management is DISABLED ***\n");
         curses_formatted_printf(out, "  The cluster will not attempt to start, stop or recover services\n");
         return pcmk_rc_ok;
-    } else if (pcmk_is_set(flags, pe_flag_stop_everything)) {
```

Comment:
first edit in 2023 in this file - right? again not checked if it comes later ...